FRINXio/frinx-frontend
# common node search

* add/remove/clear nodes to common node search

* common node search functionality

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FD-403` 
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  |


HOW TO TEST:

you need to run it against inventory server master branch, but with instructions from https://github.com/FRINXio/frinx-inventory-server/pull/229

run against .12 and open Topology discovery screen. by holding left or right shift + click on the nodes you should be able to add/remove nodes to react state, which will be later used for sending request to topology discovery api.
